src/main/java/net/mcreator/element/types/Item.java

```diff
@@ -192,6 +196,23 @@ public boolean hasEatResultItem() {
 		return isFood && eatResultItem != null && !eatResultItem.isEmpty();
 	}
 
+	public boolean hasBannerPatterns() {
+		return !isMusicDisc && !providedBannerPatterns.isEmpty();
+	}
+
+	public String getItemDescription() {
+		if (isMusicDisc) {

```

Comment:
If I understand correctly, this is used in 1.21.1 to generate item description. Wouldn't it be possible to just append music disc description at the ned of a joined string of banner pattern names to allow for both descriptions to work in 1.21.1 too as a workaround?

Reply:
I originally disabled this because music disc and patterns shared the same key, so they would appear twice. Found a more convenient solution though